digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/ChangeApi.scala

```diff
@@ -138,6 +141,76 @@ class ChangeApi extends ScalatraServlet with JacksonJsonSupport with Authenticat
         }
     )
 
+
+  def mapValue(optValue: Option[Value]): Seq[Map[String, Any]] = {
+    optValue match {
+      case Some(value) =>
+          value.asInstanceOf[Prohibitions].prohibitions.map { prohibitionValue =>
+          Map("typeId" -> prohibitionValue.typeId,
+            "exceptions" -> prohibitionValue.exceptions,
+            "validityPeriod" ->
+              prohibitionValue.validityPeriods.map { validityPeriod =>
+                Map(
+                  "startHour" -> validityPeriod.startHour,
+                  "endHour" -> validityPeriod.endHour,
+                  "startMinute" -> validityPeriod.startMinute,
+                  "endMinute" -> validityPeriod.endMinute,
+                  "days" -> validityPeriod.days.value)
+              }
+          )
+        }
+      case _ => Seq()
+    }
+  }
+
+  private def prohibitionToGeoJson(since: DateTime, changedLinearAssets: Seq[ChangedLinearAsset]) =
```

Comment:
Once  prohibitionToGeoJson and linearAssetsToGeoJson have the same main structure only the map value difference, could be create a mapValue for linear and passing the parameter.

for inear -> linearAssetsToGeoJson(..., linearMapValue)
for prohibition -> linearAssetsToGeoJson(..., prohibitionMapValue)